lib/response.js

```diff
@@ -718,6 +718,7 @@ res.header = function header(field, val) {
 
     // add charset to content-type
     if (field.toLowerCase() === 'content-type' && !charsetRegExp.test(value)) {
+      if (typeof value !== 'string')  throw "Please provide a string for content-type.";
```

Comment:
Perhaps something like "value argument must be a string for \\"content-type\\" field" or something.